packages/storybook/config/storybook/story.config.js

```diff
@@ -4,20 +4,22 @@ let outputChangedComponentPackages;
 
 const getChangedPackageStories = () => {
     try {
-        outputChangedComponentPackages = execSync('npx lerna ls --since origin/master --json --include-dependencies');
+        outputChangedComponentPackages = execSync('npx turbo run build --filter=[origin/master] --dry=json', {
+            cwd: '../..'
+        });
     } catch (error) {
         console.info('No changed component packages found.');
         process.exit(0);
     }
 
     const changedPackagesArray = JSON.parse(outputChangedComponentPackages.toString());
 
-    const changedComponentPackages = changedPackagesArray
-        .filter(pkg => pkg.location.match(new RegExp('packages/components')));
+    const changedComponentPackages = changedPackagesArray.tasks
+        .filter(pkg => pkg.directory.match(new RegExp('packages/components')));
 
     const storyPaths = [];
 
-    changedComponentPackages.forEach(pkg => storyPaths.push(`${pkg.location}/stories/*.stories.@(js|mdx)`));
+    changedComponentPackages.forEach(pkg => storyPaths.push(`../../../../${pkg.directory}/stories/*.stories.@(js|mdx)`));
```

Comment:
Is this part guaranteed to work everywhere `../../../../${pkg.directory}/stories/*.stories.@(js|mdx)`? 
Would something like `__dirname` be helpful?

Reply:
Yep, this is command is relative to this file, so the path won't change 👍🏼